database/schema.json

```diff
@@ -0,0 +1,86 @@
+{
+    "collections": [
+      {
+        "name": "Activities",
+        "fields": [
+          { "name": "activityId", "type": "Document ID" },
+          { "name": "description", "type": "String" },
+          { "name": "tickets", "type": "Number" }
+        ]
+      },
+      {
+        "name": "Prizes",
+        "fields": [
+          { "name": "prizeId", "type": "Document ID" },
+          { "name": "name", "type": "String" },
+          { "name": "ticketCost", "type": "Number" }
+        ]
+      },
+      {
+        "name": "Users",
+        "fields": [
+          { "name": "userId", "type": "Document ID" },
+          { "name": "email", "type": "String" },
+          { "name": "displayName", "type": "String" },
+          { "name": "totalTickets", "type": "Number" }
```

Comment:
the naming of this field is unclear to me. is this remaining tickets? 

i think for number of tickets earned it's easy enough to compute the value given the number of UserActivities entries

Reply:
yeah i think we should just compute `totalTickets` on the fly and not store it in the db

Reply:
is it worth keeping track of tickets remaining? might be easier to keep track of that instead of having to calculate total tickets - sum(raffle tickets used)